test/libyul/yulOptimizerTests/fullSuite/abi_example1.yul

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
Not really sure how to review this file...

Reply:
The point of `fullSuite` is not necessarily correctness (that is tested in the individual steps) but rather checking that the result does not miss any optimizations.